test/symlink.js

```diff
@@ -704,62 +706,78 @@ describe('symlink stream', function() {
   it('emits an end event', function(done) {
     var symlinkStream = vfs.symlink(outputBase);
 
-    symlinkStream.on('end', done);
+    var ended = false;
+    function end() {
+      ended = true;
+    }
+
+    symlinkStream.on('end', function() {
+      expect(ended).toBeTruthy();
+      done();
+    });
 
     var file = new File({
       base: inputBase,
       path: inputPath,
       contents: null,
     });
 
-    pipe([
-      from.obj([file]),
+    pipeline([
+      from([file]),
       symlinkStream,
-    ]);
+    ], end);
   });
 
   it('emits a finish event', function(done) {
     var symlinkStream = vfs.symlink(outputBase);
 
-    symlinkStream.on('finish', done);
+    var finished = false;
+    symlinkStream.on('finish', function() {
+      finished = true;
+    });
+
+    function finish() {
+      expect(finished).toBeTruthy();
+      done();
+    }
```

Comment:
same as above

Reply:
`finish` event is called before `pipeline`'s last callback.
(`symlinkStream:finish -> finish()`)